app/services/requests_total_items_service.rb

```diff
@@ -0,0 +1,41 @@
+class RequestsTotalItemsService
+  def initialize(requests:)
+    @requests = requests
+  end
+
+  def calculate
+    return unless requests
+
+    request_items_array = []
+    items_names = Item.where(id: request_items_ids).as_json(only: [:id, :name])
+
+    request_items.each do |items|
+      items.each do |json|
+        request_items_array << [item_name(items_names, json['item_id']), json['quantity']]
+      end
+    end
```

Comment:
As mentioned by Sean, we had a hotfix that needed to go out. https://github.com/rubyforgood/diaper/pull/1962.

In summary, the issue was that some `item_id` specified didn't match any Item. The result was not found error (which wouldn't occur here as you implemented it). However to keep the hotfix and this code aligned in approach, could we implement this so that the unknown items go filed under *Unknown Item* - https://github.com/rubyforgood/diaper/pull/1962/files#diff-cd57d2a9133000ab8443d9eb0c8a78c9R55